File: CS/DXApplication14/ApptViewModel.cs
using DevExpress.Mvvm;
using System;

namespace DXApplication14
{
    public class ApptViewModel : BindableBase {
        public string Subject {
            get { return GetValue<string>(); }
            set { SetValue(value); }
        }
        public DateTime Start {
            get { return GetValue<DateTime>(); }
            set { SetValue(value); }
        }
        public DateTime End {
            get { return GetValue<DateTime>(); }
            set { SetValue(value); }
        }
        public string Description {
            get { return GetValue<string>(); }
            set { SetValue(value); }
        }
        public int Type {
            get { return GetValue<int>(); }
            set { SetValue(value); }
        }
        public string RecurrenceInfo {
            get { return GetValue<string>(); }
            set { SetValue(value); }
        }
    }
}

File: VB/DXApplication14/ApptViewModel.vb
Imports DevExpress.Mvvm

Namespace DXApplication14

    Public Class ApptViewModel
        Inherits BindableBase

        Public Property Subject As String
            Get
                Return GetValue(Of String)()
            End Get

            Set(ByVal value As String)
                SetValue(value)
            End Set
        End Property

        Public Property Start As Date
            Get
                Return GetValue(Of Date)()
            End Get

            Set(ByVal value As Date)
                SetValue(value)
            End Set
        End Property

        Public Property [End] As Date
            Get
                Return GetValue(Of Date)()
            End Get

            Set(ByVal value As Date)
                SetValue(value)
            End Set
        End Property

        Public Property Description As String
            Get
                Return GetValue(Of String)()
            End Get

            Set(ByVal value As String)
                SetValue(value)
            End Set
        End Property

        Public Property Type As Integer
            Get
                Return GetValue(Of Integer)()
            End Get

            Set(ByVal value As Integer)
                SetValue(value)
            End Set
        End Property

        Public Property RecurrenceInfo As String
            Get
                Return GetValue(Of String)()
            End Get

            Set(ByVal value As String)
                SetValue(value)
            End Set
        End Property
    End Class
End Namespace
